route/option.go

```diff
@@ -200,9 +210,16 @@ PLEASE NOTE: When defining a custom TravelTimeMeasure, this measure must not
 account for any service times. To account for services times please use the
 Services option.
 */
-func TravelTimeMeasures(timeMeasures []ByIndex) Option {
+func TravelTimeMeasures(timeMeasures any) Option {
```

Comment:
AFAICT we use `any` here to not break the api, right? Why don't we introduce a new option called that is type safe:
* `TravelTimeMeasures([]ByIndex) Option`
* `TravelTimeDependentMeasures([]DependentByIndex) Option`
Same applies to `ValueFunctionMeasures`. That would leave everything backwards compatible and we would not loose any compiler safety.